src/widget/wtracktableview.cpp

```diff
@@ -840,16 +905,41 @@ void WTrackTableView::keyPressEvent(QKeyEvent* event) {
                 m_pTrackMenu->slotShowDlgTrackInfo();
             }
         }
-    } break;
+        return;
+    }
     case kHideRemoveShortcutKey: {
         if (event->modifiers() == kHideRemoveShortcutModifier) {
             hideOrRemoveSelectedTracks();
-            return;
         }
-    } break;
+        return;
+    }
     default:
-        QTableView::keyPressEvent(event);
+        break;
+    }
+    TrackModel* trackModel = getTrackModel();
+    if (trackModel && !trackModel->isLocked()) {
+        if (event->matches(QKeySequence::Delete) || event->key() == Qt::Key_Backspace) {
```

Comment:
Backspace is used to navigate back in NEMO. It is sup-prizing that the track is deleted without a warning. 
Can we remove it here?

Reply:
Yes. Also I see that ctrl->backspace also deleted the track (with confirmation popup). It surprised me I missed that.

Reply:
Removed. Please resolve this conversation, @daschuer